Stack/Python/minimum_ele.py

```diff
@@ -0,0 +1,201 @@
+            if i==data:
+                continue
+            temp.append(i)
+        stack.clear()
+        for i in temp:
+            insert_stk(stack,i)
+        for i in support_stack:
+            if i==data:
+                continue
+            temp1.append(i)
+        support_stack.clear()
+        for i in temp1:
+            insert_stk(support_stack,i)
+        return 1
+    else:
+        return -100000
+
+def result():
+    for i in range(len(stack)):
+        print(stack[i],end=" ")
+    print()
+   # for i in range(len(support_stack)):
+    #    print(support_stack[i],end=" ")
+
+def get_min():
+    global support_stack
+    if len(support_stack)>0:
+        return support_stack[0]
+    else:
+        return -1
+
+
+if __name__=="__main__":
+    flag=1
+    t_supp=-1
+    t_main=-1
+    yon=input('Wanna create a stack?(y/n)')
+    if yon=='y':
+        while(flag!=0):
+            print("Menu:")
+            print("0.Quit process")
+            print("1.Push into stack")
+            print("2.Delete from stack")
+            print("3.Print data")
+            print("4.Get minimum")
+            flag=int(input())
+            if flag==1:
+                ele=int(input('Enter data: '))
+                insert_stk(stack,ele)
+                t_main+=1
+                if t_supp>=0:
+                    f=0
+                    for i in range(len(support_stack)):
+                        if ele < support_stack[i]:
+                            f=1
+                            support_stack.insert(i,ele)
+                            break
+                    t_supp+= 1
+                    if f!=1:
+                        insert_stk(support_stack,ele)
+
+                elif t_supp==-1:
+                    t_supp+=1
+                    insert_stk(support_stack,ele)
+            elif flag==2:
+                d=int(input('Data to be deleted: '))
+                y=remove(d)
+                if y==-100000:
+                    t_main-=1
+                    print('Empty stack!')
+                else:
+                    print('Deleted!!')
+            elif flag==3:
+                result()
+                print()
+            elif flag==4:
+                ans=get_min()
+                if ans!=-1:
+                    print("So minimum value inside stack is: ",end="")
+                    print(ans)
+                else:
+                    print("Oops! stack is empty")
+    print('Okay bye there!')
+
+
+#Time comlpexity: O(1) ---->inserting an element
+#                 O(1) ---->poping an element ,O(n) for deleting any particuler element
+#                 O(1)  --->getting minimum element from stack
+
+#Space Complexity : O(n)
+
+#e.g
+#Wanna create a stack?(y/n)y
+#Menu:
+#0.Quit process
+#1.Push into stack
+#2.Delete from stack
+#3.Print data
+#4.Get minimum
+#1
+#Enter data: 11
+#Menu:
+#0.Quit process
+#1.Push into stack
+#2.Delete from stack
+#3.Print data
+#4.Get minimum
+#1
+#Enter data: 12
+#Menu:
```

Comment:
Do Proper Editing here, Give Proper Space and Tabs Here, Make it Readable
Do this Everywhere in this Comments

Reply:
DO this changes @Aarti002